src/views/PurchaseOrder.vue

```diff
@@ -107,21 +107,21 @@ export default defineComponent({
     methods: {
       async parse(event) {
         const file = event.target.files[0];
-        if(file){
-          this.file = file;
-          this.content = await this.parseCsv(this.file);
-          this.fileColumns = Object.keys(this.content[0]);
-          showToast(translate("File uploaded successfully"));
-        } else {
-          showToast(translate("No new file upload. Please try again"));
+        try {
+          if (file) {
+            this.file = file;
+            this.content = await this.parseCsv(this.file);
+            this.fileColumns = Object.keys(this.content[0]);
+            showToast(translate("File uploaded successfully"));
+          } else {
+            showToast(translate("No new file upload. Please try again"));
+          }
+        } catch {
+          this.file = this.content = {}
```

Comment:
Why we are making file empty? Also this.content should be an array as we are checking for its length.

Reply:
Yes, sir. Making `this.file` empty is not required. Also, updated `this.content` to be an array now.